libvirt.go

```diff
@@ -373,6 +374,7 @@ func domainXML(d *Driver) (string, error) {
 		CacheMode:  d.CacheMode,
 		IOMode:     d.IOMode,
 		DiskPath:   d.getDiskPath(),
+		SerialPath: fmt.Sprintf("/var/log/libvirt/qemu/%s-serial.log", d.MachineName),
```

Comment:
@guillaumerose what if this directory doesn't exist, does the libvirt domain creates during `creating/running` the domain or we are just setting the default value here and in the crc codebase actually use a different path for the file?

Reply:
yes very good question. CRC use this path. If it can't create the file, the VM doesn't start.
Maybe @cfergeau will know if this directory always exists?

Reply:
Good question, this is where the qemu driver puts the VM logs, so it should exist. I don't know if it's possible to change this dir to something different at compile time or through a config file. So I'd just assume it's there.
By «  I can't put it in the machine directory », is this because it's owned by root, or for other reasons? If it's because it's owned by root, I would say it may be acceptable as the file can be removed when needed (crc delete), it's just reading/writing to it which wouldn't work.

Reply:
> is this because it's owned by root, or for other reasons? 

yes

so, let's try this ?

Reply:
Dunno what is "this", but fine with me in either location. `/var/log/libvirt` is only accessible to root so it's harder to get the log file.

Reply:
An alternative is to use 
```
diff --git a/pkg/libvirt/domain.go b/pkg/libvirt/domain.go
index 28f9a71..ccea90b 100644
--- a/pkg/libvirt/domain.go
+++ b/pkg/libvirt/domain.go
@@ -75,7 +75,11 @@ func domainXML(d *Driver, machineType string) (string, error) {
                                },
                        },
                        Consoles: []libvirtxml.DomainConsole{
-                               {},
+                               {
+                                       Source: &libvirtxml.DomainChardevSource {
+                                               StdIO: &libvirtxml.DomainChardevSourceStdIO{},
+                                       },
+                               },
                        },
                        RNGs: []libvirtxml.DomainRNG{
                                {
```
which will put the console output to `/var/log/libvirt/qemu/crc.log` which is managed by libvirt. Accessing this log will require root.

Reply:
Oh great! it's better as it find itself where to put the file.